### DISCUSSION TITLE
Add step to CPG ArrayRangeExpression

### DISCUSSION THREAD

**Discussion by maximiliankaul** [Fraunhofer-AISEC/cpg#372] [Ideas]
Python supports `step` for slices. Currently, the python-jep branch uses `ArrayRangeExpression` for slices, which provide `floor` and `ceiling`.
Suggestion: add `step` to `ArrayRangeExpression`

**Comment by oxisto** [comment]
I think that would be ok

**Reply by maximiliankaul** [reply]
oxisto: EOG should be made aware of this expression.
